unity-webgl-output/Build/unity-webgl-output.framework.js

```diff
@@ -0,0 +1,15968 @@
+function unityFramework(Module) {
```

Comment:
This file is the deminified version of the `.gz` file and is included to help debug the networking logic that was added with the `cdmeditor.jslib` file